lib/create-space-api.js

```diff
@@ -57,6 +57,18 @@ import entities from './entities'
  * @prop {function} getContentTypeSnapshots
  */
 
+function raiseDeprecationWarning (method) {
+  console.warn(
+    [
+      `Deprecated: Space.${method}() will be removed in version 6.0.0.`,
```

Comment:
Let's change this to future `major versions`

Reply:
done